install/rdtclient-install.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2023 tteck
+# Author: tteck (tteckster)
+# License: MIT
+# https://github.com/tteck/Proxmox/raw/main/LICENSE
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y curl
+$STD apt-get install -y sudo
+$STD apt-get install -y mc
+$STD apt-get install -y unzip
+msg_ok "Installed Dependencies"
+
+msg_info "Installing rdtclient"
+wget -q https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb
+dpkg -i packages-microsoft-prod.deb &>/dev/null
```

Comment:
$STD dpkg -i packages-microsoft-prod.deb

Reply:
I actually had that before and it failed on me (didn't silence anything), that's why I went with &>/dev/null, I'll try it again

Reply:
I've setup my own branch (with proper changes on the repo urls) to test this directly on my proxmox's server, to simulate a real installation and it didn't silence those commands, worry not, I'll give it a spin later